client/src/SignupNonProfit.tsx

```diff
@@ -59,129 +101,184 @@ function SignupNonProfit() {
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="state"
+                                        label="State"
+                                        placeholder="State"
+                                        value={formData.state}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="ein"
+                                        label="Entity Identification Number (EIN)"
+                                        placeholder="EIN"
+                                        value={formData.ein}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="tax_exempt_id"
+                                        label="Tax Exempt ID"
+                                        placeholder="Tax Exempt ID"
+                                        value={formData.tax_exempt_id}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={8} xs={12}>
+                                    <Select
+                                        id="nonprofit_classification"
+                                        label="IRS Nonprofit Organization Classification"
+                                        placeholder="Select classification"
+                                        options={exemptions}
+                                        value={formData.nonprofit_classification}
+                                        onChange={handleSelectChange}
+                                    />
+                                </Grid>
+                                    <Grid item md={8} xs={12}>
+                                            <Button 
+                                                onClick={handleNextClick}
+                                                className={classes.button}
+                                                fullWidth
+                                                >
+                                                Next
+                                            </Button>
+                                    </Grid> 
+                                </Grid>
+                                
+                        ) : (
+                            <Grid container spacing={5}>
+                                <Grid item xs={12}>
+                                    <Typography component="p" align="left">
```

Comment:
can we get a button or link looking button, something minimal, right above this that does `back` to page 1